src/openms/source/ANALYSIS/OPENSWATH/DIAScoring.cpp

```diff
@@ -362,14 +388,14 @@ namespace OpenMS
       }
       if (ratio > max_ratio) {max_ratio = ratio;}
 
-      double ddiff_ppm = std::fabs(mz - (mono_mz - 1.0 / (double) ch)) * 1000000 / mono_mz;
+      double ddiff_ppm = std::fabs(mz - (mono_mz - 1.0 / (double) ch)) * 1e6 / mono_mz;
```

Comment:
is this correct? why 1.0? but doesn't seem to have a lot of impact

Reply:
I'll change to C13C12 diff right?